File: Iniciante/1117.py
nota_valida = 0
media = 0
while nota_valida != 2:
    nota = float(input())
    if nota < 0 or nota > 10:
        print('nota invalida')
    else:
        media += nota / 2
        nota_valida += 1
print('media = {:.2f}'.format(media))

File: Iniciante/1117.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    int contadorPositivas;
    double notas, soma;
    while (contadorPositivas != 2){
        cin >> notas;
        if (notas >= 0 && notas <= 10){
            soma += notas;
            contadorPositivas++;
        } else {
            cout << "nota invalida" << endl;
        }
    }
    double media = soma / 2;
    cout << fixed << setprecision(2);
    cout << "media = " << media << endl;
    return 0;
}